plugins/hcp/commands/cluster-health-check.md

```
@@ -0,0 +1,712 @@
+No critical issues found. Cluster is operating normally.
+
+DIAGNOSTIC COMMANDS:
+Run these commands for detailed information:
+
+kubectl get hostedcluster my-cluster -n clusters -o yaml
+kubectl get pods -n clusters-my-cluster
+kubectl get nodepool -n clusters
+```
+
+Output, with warnings:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)